### ISSUE TITLE
Unobserved exception in CSharpCompletionTextEditorExtension

### ISSUE THREAD

**Issue by Therzok** [mono/monodevelop#7049]
```
System.AggregateException: One or more errors occurred. (A task was canceled.) ---> System.Threading.Tasks.TaskCanceledException: A task was canceled.
  --- End of inner exception stack trace ---
  at System.Threading.Tasks.Task.ThrowIfExceptional (System.Boolean includeTaskCanceledExceptions) [0x00011] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-08/external/bockbuild/builds/mono-x64/external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/Task.cs:2051
  at System.Threading.Tasks.Task`1[TResult].GetResultCore (System.Boolean waitCompletionNotification) [0x0002b] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-08/external/bockbuild/builds/mono-x64/external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/Future.cs:506
  at System.Threading.Tasks.Task`1[TResult].get_Result () [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-08/external/bockbuild/builds/mono-x64/external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/Future.cs:476
  at MonoDevelop.CSharp.Completion.CSharpCompletionTextEditorExtension+<>c.<HandleCodeCompletionAsync>b__30_0 (System.Threading.Tasks.Task`1[TResult] t) [0x00001] in /Users/therzok/Work/md/monodevelop/main/src/addins/CSharpBinding/MonoDevelop.CSharp.Completion/CSharpCompletionTextEditorExtension.cs:225
  at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2[TAntecedentResult,TResult].InnerInvoke () [0x00024] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-08/external/bockbuild/builds/mono-x64/external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/TaskContinuation.cs:206
  at System.Threading.Tasks.Task.Execute () [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-08/external/bockbuild/builds/mono-x64/external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/Task.cs:2343
```

> VS bug [#798074](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/798074)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-19] Comment by **xamarinc**



Fixed in version 8.4.0.196 (master)



Pull Request #8033 merged by: Marius Ungureanu  
 Author: mono  
 Commit: 81abd7ce32504e3fdcdfc004069aeeb375249733 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-21] Comment by **xamarinc**



Fixed in version 8.3.0.592 (release-8.3)



Pull Request #8033 merged by: Marius Ungureanu  
 Author: mono  
 Commit: 81abd7ce32504e3fdcdfc004069aeeb375249733 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-22] Comment by **xamarinc**



Fixed in version 8.2.0.1063 (release-8.2)



Pull Request #8072 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 0ca1979ece0fd8b09240e6a5f4ab089b9fc88471 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-24] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #8033 merged by: Marius Ungureanu  
 Author: mono  
 Commit: 81abd7ce32504e3fdcdfc004069aeeb375249733 (xamarin/monodevelop)
